test/unit/specs/components/common/TmSessionImport.spec.js

```diff
@@ -16,7 +16,8 @@ describe(`TmSessionImport`, () => {
   beforeEach(() => {
     store = new Vuex.Store({
```

Comment:
We usually just mock the store without using the actual Vuex store. This might be still old in here. Checkout https://github.com/luniehq/lunie/blob/develop/test/unit/specs/components/ActionModal/components/SendModal.spec.js#L33

Reply:
Yeah I like the new way better too. That was just there already. I'll update it 👍